pkgs/top-level/aliases.nix

```diff
@@ -1320,6 +1320,7 @@ mapAliases ({
   rng_tools = throw "'rng_tools' has been renamed to/replaced by 'rng-tools'"; # Converted to throw 2022-02-22
   robomongo = throw "'robomongo' has been renamed to/replaced by 'robo3t'"; # Converted to throw 2022-02-22
   rockbox_utility = rockbox-utility; # Added 2022-03-17
+  rocm-llvm-llvm = llvmPackages_rocm.llvm; # For update script
```

Comment:
Is this required for the updater? It should not rely on aliases.

Reply:
I couldn't find any other sane way to do it, so yes.
Is there any difference between a package directly referencing another without any change and an alias?

Reply:
I used an alias because it would have just been the same thing in ``all-packages.nix``, so that's why I'm asking.

Reply:
I see the confusion now.
It's for THIS update script, the generic one is fine.
``nix-shell maintainers/scripts/update.nix --argstr commit true --argstr keep-going true --arg predicate '(path: pkg: builtins.elem (pkg.pname or null) [ "clang-ocl" "composable_kernel" "hipcub" "hipsparse" "miopen" "miopengemm" "rccl" "rocblas" "rocclr" "rocfft" "rocm-cmake" "rocm-comgr" "rocm-device-libs" "rocm-opencl-runtime" "rocm-runtime" "rocm-smi" "rocm-thunk" "rocminfo" "rocmlir" "rocprim" "rocrand" "rocsparse" "rocthrust" "rocwmma" "tensile" "hip-amd" "hip-common" "hipcc-amd" "rocprofiler" "roctracer" "rocsolver" "rocalution" "rocm-llvm-llvm" ])'``

Reply:
What happens otherwise is it isn't included when updating.